doc/lsst.ap.verify/limitations.rst

```diff
@@ -0,0 +1,12 @@
+.. py:currentmodule:: lsst.ap.verify
+
+.. _ap-verify-request:
+
+########################################
+Known Limitations 
+########################################
```

Comment:
Sphinx requires that the header symbol lines be exactly as long as the header text. Please do run `package-docs build`, it will catch such things.